EIPS/eip-action_standard.md

```diff
@@ -0,0 +1,301 @@
+---
+eip:
+title: Token Interaction Standard
+description: A standard action messaging protol for interactions on and between NFTs
+author: Alexi (@0xalxi)
+discussions-to:
+type: Standards Track
+category: ERC
+status: Draft
+created: 2021-4-18
+---
+
+## Simple Summary
+
+A standard messaging protocol for interactive tokens.
+
+## Abstract
+
+This standard defines a broadly applicable action messaging protocol for the transmission of arbitrary, user-initiated actions between tokens. Modular statefulness is achieved with parameterized state contracts that can provide arbitration and settlement of the action process.
+
+## Motivation
+
```

Comment:
Might be worth adding a _short_ concrete motivating example that shows some tokens and their actions, and how this is a problem today.